game-core/src/main/java/games/strategy/util/PointFileReaderWriter.java

```diff
@@ -105,32 +97,36 @@ public static void writeOneToManyPolygons(final OutputStream sink, final Map<Str
     checkNotNull(sink);
     checkNotNull(mapping);
 
-    final StringBuilder out = new StringBuilder();
-    final Iterator<String> keyIter = mapping.keySet().iterator();
-    while (keyIter.hasNext()) {
-      final String name = keyIter.next();
-      out.append(name).append(" ");
-      final List<Polygon> points = mapping.get(name);
-      for (Polygon polygon : points) {
-        out.append(" < ");
-        for (int i = 0; i < polygon.npoints; i++) {
-          out.append(" (").append(polygon.xpoints[i]).append(",").append(polygon.ypoints[i]).append(")");
-        }
-        out.append(" > ");
-      }
-      if (keyIter.hasNext()) {
-        out.append("\r\n");
-      }
-    }
-    write(out, sink);
+    write(mapping.entrySet().stream()
+        .map(entry -> entry.getKey() + " " + entry.getValue().stream()
+            .map(PointFileReaderWriter::polygonToString)
+            .collect(Collectors.joining()))
+        .collect(Collectors.joining("\r\n")), sink);
   }
 
-  private static void write(final StringBuilder buf, final OutputStream sink) throws IOException {
+  private static void write(final String string, final OutputStream sink) throws IOException {
     try (Writer out = new OutputStreamWriter(new CloseShieldOutputStream(sink), StandardCharsets.UTF_8)) {
-      out.write(buf.toString());
+      out.write(string);
     }
   }
 
+  private static String pointsToString(final List<Point> points) {
```

Comment:
nit, first usage is around line 140, private methods should follow after their first usage, please move to be below the calling method.